### ISSUE TITLE
Add missing accessibility labels and hints

### ISSUE THREAD

**Issue by mhoran** [mhoran/weechatRN#105]
WeechatRN should be accessible. Right now only a few accessibility hints have been added as needed for tests. All interactive components should be labeled and any confusing interactions should have accessibility hints.

For the buffer lines, VoiceOver reads the line left to right, including the nick, message, and timestamp. Whatever is done here needs to make sense.
